test/intl_suggestors/utils_test.dart

```diff
@@ -87,33 +86,44 @@ void main() {
 
         final parsedInterpolation = parsedExpression as StringInterpolation;
         expect(parsedInterpolation.isMultiline, isMultiline);
-
         final testResult = IntlMessages('Test')
             .syntax
             .functionDefinition(parsedInterpolation, 'Namespace', "NamePrefix");
         expect(testResult, expectedResult);
       }
 
+      void ignoreSingleLineAndMultiline(
+          String testStr, bool isMultiline, String expectedResult) async {
+        final parsedExpression = await sharedContext.parseExpression(testStr);
+        expect(parsedExpression is StringInterpolation, isTrue);
+        var parsedInterpolation = parsedExpression as StringInterpolation;
+        expect(parsedInterpolation.isMultiline, isMultiline);
+        var data = textFromInterpolation(parsedInterpolation);
+        if (data.isEmpty) {
+          data = expectedResult;
```

Comment:
Is this data variable used anywhere else in the function? it doesn't seem to need to be reset here.

Reply:
Updated the logic here for `data` variable